src/widget/widget.cpp

```diff
@@ -675,6 +675,37 @@ void Widget::onSelfAvatarLoaded(const QPixmap& pic)
     profilePicture->setPixmap(pic);
 }
 
+void Widget::onCoreChanged(Core& core) {
```

Comment:
brace should be on the next line, was this formatted by clang-format this way?

Reply:
It wasn't, I forgot to run format. Fixed.